test/gtest/ucp/test_ucp_sockaddr.cc

```diff
@@ -758,6 +769,50 @@ UCS_TEST_P(test_ucp_sockaddr, err_handle) {
     EXPECT_EQ(1u, sender().get_err_num());
 }
 
+UCS_TEST_SKIP_COND_P(test_ucp_sockaddr, listener_invalid_params,
+                     nonparameterized_test())
+{
+    ucp_listener_params_t params;
+    ucp_listener_h listener;
+    ucs_status_t status;
+
+    params.field_mask = 0;
+    /* address and conn/accept handlers are not specified */
+    status = create_listener_wrap_err(params, listener);
+    ASSERT_EQ(UCS_ERR_INVALID_PARAM, status);
+
+    /* add listen address, use ANY addr to avoid BUSY error in the end */
+    m_test_addr.reset_to_any();
+    params.field_mask      |= UCP_LISTENER_PARAM_FIELD_SOCK_ADDR;
+    params.sockaddr.addr    = m_test_addr.get_sock_addr_ptr();
+    params.sockaddr.addrlen = m_test_addr.get_addr_size();
+    /* accept handlers aren't set */
+    status = create_listener_wrap_err(params, listener);
+    ASSERT_EQ(UCS_ERR_INVALID_PARAM, status);
+
+    /* define conn handler to NULL */
+    params.field_mask      |= UCP_LISTENER_PARAM_FIELD_CONN_HANDLER;
+    params.conn_handler.cb  = NULL;
+    params.conn_handler.arg = NULL;
+    status = create_listener_wrap_err(params, listener);
+    ASSERT_EQ(UCS_ERR_INVALID_PARAM, status);
+
+    /* define accept handler to NULL */
+    params.field_mask        |= UCP_LISTENER_PARAM_FIELD_ACCEPT_HANDLER;
+    params.accept_handler.cb  = NULL;
+    params.accept_handler.arg = NULL;
+    status = create_listener_wrap_err(params, listener);
+    ASSERT_EQ(UCS_ERR_INVALID_PARAM, status);
```

Comment:
need to re-set field mask for every test.
and also test with non-NULL cb/arg pointers - to test that combination of 2 handlers, or no handlers at all, causes INVALID_PARAM. and not because the cb is NULL